Signal/src/Models/TSMessageAdapaters/JSQMediaItem+OWS.m

```diff
@@ -32,4 +38,33 @@ - (CGSize)ows_adjustBubbleSize:(CGSize)bubbleSize forImage:(UIImage *)image {
     return bubbleSize;
 }
 
+- (CGSize)sizeOfImageAtURL:(NSURL *_Nullable)imageURL
```

Comment:
Premise makes sense.

When would imageURL be nil? Seems like a callers concern.